packages/osd-ui-shared-deps/index.d.ts

```diff
@@ -44,29 +44,14 @@ export const jsFilename: string;
 export const jsDepFilenames: string[];
 
 /**
- * Filename of the unthemed css file in the distributable directory
+ * Re-export all types from theme_config
  */
-export const baseCssDistFilename: string;
+export * from './theme_config';
 
 /**
- * Filename of the dark-theme css file in the distributable directory
- */
-export const darkCssDistFilename: string;
-
-/**
- * Filename of the dark-theme css file in the distributable directory
- */
-export const darkV8CssDistFilename: string;
-
-/**
- * Filename of the light-theme css file in the distributable directory
- */
-export const lightCssDistFilename: string;
```

Comment:
These are part of our public API. If we intend to keep these changes only in 3.0.0, removing these would be fine but if we intend to backport this PR, we will need to continue to provide these and mark them as deprecated.

Reply:
Is using jsdoc good enough for marking deprecated?

Reply:
What we do is add `@deprecated` here and call it out in the CHANGELOG.